WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -564,6 +564,8 @@ public void onResponse(JSONObject jsonObject) {
                     }
                 }, null);
             } else {
+                // Clear the URL on successful login so it's not persisted the next time we add a self-hosted site
+                mUrlEditText.setText("");
```

Comment:
I think we should call this even if `isWPComLogin` (in case the user entered his wpcom url).